packages/svelte-check/src/index.ts

```diff
@@ -149,8 +149,12 @@ class DiagnosticsWatcher {
         filePathsToIgnore: string[],
         ignoreInitialAdd: boolean
     ) {
-        watch(`${workspaceUri.fsPath}/**/*.{svelte,d.ts,ts,js,jsx,tsx,mjs,cjs,mts,cts}`, {
-            ignored: ['node_modules', 'vite.config.{js,ts}.timestamp-*']
+        watch(workspaceUri.fsPath, {
+            ignored: [
+                'node_modules',
+                'vite.config.{js,ts}.timestamp-*',
```

Comment:
Is this correct? AFAIK `node_modules` meant that everything with node_modules in the path was ignored, right? If not, we should change that.
The vite config string also looks suspiscuous, it's still using glob syntax, does this need to turn into a regex?

Reply:
Ok this has a couple of more bugs, will fix